logic/transaction.js

```diff
@@ -15,6 +15,15 @@ var self, modules, __private = {}, genesisblock = null;
 
 __private.types = {};
 
+// Private methods
+__private.calculateFee = function (trs, sender) {
+	var fee = 0;
+	if (trs.data) {
+		fee = constants.fees.data;
+	}
+	return __private.types[trs.type].calculateFee.call(this, trs, sender) + fee || false;
```

Comment:
@4miners, I was going through the code, and I see that we are using javascript based primitive number type to calculate the fee. For example, [here](https://github.com/LiskHQ/lisk/blob/development/logic/multisignature.js#L57). I think it'd be inconsistent to use `bignumber` only here. Should we update fee calculation to use `bignumber` everywhere? What do you think?